caffe2/operators/pool_op.cu

```diff
@@ -253,7 +254,7 @@ __global__ void AveragePool3DForwardNHWCCUDAKernel(
   const int xh = yh * stride_h;
   const int xw = yw * stride_w;
   const int p = max(xd - pad_p, 0);
-  const int a = min(xd - pad_p + kernel_h, X_D);
+  const int a = min(xd - pad_p + kernel_d, X_D);
```

Comment:
We really need to improve our test coverage on ops.

Reply:
Sure, I will update pooling_test later.